App.js

```diff
@@ -41,37 +40,30 @@ module.exports = function App() {
 
     return (
         <GestureHandlerRootView style={{ flex: 1 }}>
-            <KeyboardAvoidingView
-                behavior={Platform.OS === 'ios' ? 'padding' : null}
-                style={{ flex: 1 }}
-            >
-                <SafeAreaView style={{ flex: 1 }}>
```

Comment:
It's still useful in general, but the reason I got rid of it at the `App.js` level is because the padding to account for the Home Bar would appear as white. We'll want to make sure the `SafeAreaView` is always the same color as our background, which is tough to do at this level.